Assets/MixedRealityToolkit/_Core/Utilities/Async/Internal/AsyncCoroutineRunner.cs

```diff
@@ -28,11 +30,13 @@ namespace Microsoft.MixedReality.Toolkit.Core.Utilities.Async.Internal
     /// This Async Coroutine Runner is just a helper object to
     /// ensure that coroutines run properly with async/await.
     /// </summary>
-    public class AsyncCoroutineRunner : MonoBehaviour
+    internal sealed class AsyncCoroutineRunner : MonoBehaviour
```

Comment:
why is this no longer public? wouldn't customer code benefit from this class?

Reply:
Because nothing outside of this assembly should be calling on this.
No one should be using this class except for the utilities. That's why it's in the internal folder.

Reply:
wouldn't there be customer benefit from having a solid / deadlock safe async runner available?

Reply:
Yes and that's exactly what they get with this change.

Reply:
Should have been more clear in the question... Wouldn't it benefit customer code to be able to directly use this class?

Reply:
No, as they shouldn't be directly using this class, because it's an implementation detail of the overall utility.

It's to protect the integrity of this system and its function.